migrate2rocky/migrate2rocky.sh

```diff
@@ -788,16 +848,47 @@ EOF
     	    exit_message "Can't enable modules ${enabled_modules[*]}"
     fi
 
-    # Make sure that excluded repos are disabled.
+    # Make sure that excluded modules are disabled.
     infomsg $'Disabling excluded modules\n\n'
     safednf -y module disable "${module_excludes[@]}" ||
     	exit_message "Can't disable modules ${module_excludes[*]}"
 
     infomsg $'\nSyncing packages\n\n'
     dnf -y distro-sync || exit_message "Error during distro-sync."
 
+    # Disable Stream repos.
+    if (( ${#installed_sys_stream_repos_pkgs[@]} ||
+	  ${#installed_stream_repos_pkgs[@]} )); then
+	dnf -y --enableplugin=config_manager config-manager --set-disabled \
+	    "$stream_prefix*" ||
+	    errmsg \
+$'Failed to disable CentOS Stream repos, please check and disable manually.\n'
+
+	infomsg $'\nCentOS Stream Migration Notes:\n\n'
+	cat <<EOF
+Because CentOS Stream leads RockyLinux by the next point release many packages
+in Stream will have higher version numbers than those in RockyLinux, some will
+even be rebased to a new upstream version.  Downgrading these packages to the
+versions in RockyLinux carries the risk that the older version may not
+recognize config files, data or other files generated by the newer version in
+Stream.
+
+To avoid issues with this the newer package versions from CentOS Stream have
+been retained.  Also the CentOS Stream repositories have been retained but
+renamed with a prefix of "stream-" to avoid clashing with RockyLinux
+repositories, but these same repos have also been disabled so that future
+package installs will come from the stock RockyLinux repositories.
+
+If you do nothing except update to the next point release of RockyLinux when it
+becomes available then the packages retained from Stream should be replaced at
+that time.  If you need to update a package from Stream (eg: to fix a bug or
+security issue) then you will need to enable the appropriate repository to do
+so.
+EOF
+    fi
+
     if rpm --quiet -q subscription-manager; then
-	infomsg $'Subscription Manager found on system.\n'
+	infomsg $'\nSubscription Manager found on system.\n\n'
```

Comment:
some comments contains
subscription-manger
instead of 
subscription-manager
e.g. in 
package_swaps()
https://github.com/rocky-linux/rocky-tools/blob/f98474c5d14937ed9660a7f06994eb19fc2725dd/migrate2rocky/migrate2rocky.sh#L688
and
https://github.com/rocky-linux/rocky-tools/blob/f98474c5d14937ed9660a7f06994eb19fc2725dd/migrate2rocky/migrate2rocky.sh#L650
see also (severely):
https://github.com/rocky-linux/rocky-tools/blob/f98474c5d14937ed9660a7f06994eb19fc2725dd/migrate2rocky/migrate2rocky.sh#L214
exercise:
https://github.com/rocky-linux/rocky-tools/blob/f98474c5d14937ed9660a7f06994eb19fc2725dd/migrate2rocky/migrate2rocky.sh#L598